src/index.html

```diff
@@ -1178,6 +1179,32 @@ <h6>Reporting</h6>
                                 </div>
                             </ul>
                         </div>
+                        <div class="tabs-panel is-active" id="citationPanel">
+                            <ul class="padding-top-large padding-bottom-large">
+                                <li><a href="#archive-data-reference">How to reference archived data?</a></li>
+                                <li><a href="#citation-reference">How to cite EVA?</a></li>
+                            </ul>
+                            <ul class="accordion" data-accordion>
+                                <div id="archive-data-reference">
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>How to reference archived data?</h5></a>
+                                        <div class="accordion-content" data-tab-content>
+                                            <p>Archived data can be referenced using the project accession & associated URL e.g. The variant data for this study have been deposited in the European Variation Archive (EVA) at EMBL-EBI under accession number PRJEBxxxx (<a>https://www.ebi.ac.uk/eva/?eva-study=PRJEB*****</a>)</p>

```

Comment:
Please break this lines so they are not that long

Reply:
updated